### DISCUSSION TITLE
Creating a HTTP/0.9 Request

### DISCUSSION THREAD

**Discussion by dan-nicholls** [curl/curl#13663] [Q&A] [answered]
I just wanted to confirm that the `--http0.9` flag is only used to accept `HTTP/0.9` **response**, rather than also making a 0.9 request?

I know that a `HTTP/0.9` request isn't hard to form but just wanted to confirm as I am seeing a `HTTP/1.1` request being sent:
```bash
curl --http0.9 -v 127.0.1.1:9000/
*   Trying 127.0.1.1:9000...
* Connected to 127.0.1.1 (127.0.1.1) port 9000 (#0)
> GET / HTTP/1.1
> Host: 127.0.1.1:9000
> User-Agent: curl/7.88.1
> Accept: */*
>
* Closing connection 0
Invalid request%
```

**Comment by jay** [comment, marked as answer]
> I just wanted to confirm that the `--http0.9` flag is only used to accept `HTTP/0.9` **response**, rather than also making a 0.9 request?

Yes that's correct, see [--http0.9](https://curl.se/docs/manpage.html#--http09) doc
